Mathlib/Topology/ContinuousOn.lean

```diff
@@ -1498,6 +1498,13 @@ lemma ContinuousOn.union_continuousAt
   (fun h => ContinuousWithinAt.continuousAt (continuousWithinAt hs h) <| IsOpen.mem_nhds s_op h)
   (ht _)
 
+/-- If a function is continuous on the union of two sets, it is continuous on both of them. -/
+theorem continuousOn_of_union {X Y : Type*} [TopologicalSpace X] [TopologicalSpace Y]
```

Comment:
Small nitpick: this file already has `α β` as topological spaces, so I think it would be cleaner to use those variables 
(and perhaps `ContinuousOn.union_continuousAt` and `ContinuousOn.union_isClosed` should also be changed in a similar manner)

Reply:
*If* you care about using Greek letters, you should file a preliminary PR renaming all the occurrences in the file. (I would certainly review such a PR.)

Reply:
Done in #22691 (just to clarify, my intention wasn't to suggest that these results *should* be stated with Greek letters, but rather that it might be nicer to stay consistent with the notation of (the majority of) the file)

Reply:
I would say they should use Roman letters (usually) - but also that changing all files manually is a lot of work that might not be the best use of one's time.

Reply:
That sounds reasonable - good to know:)